src/app/campaign/inventory/inventory.component.scss

```diff
@@ -29,3 +24,49 @@ section {
   display: flex;
   align-items: center;
 }
+
+.loading {
+  height: 80px;
+  display: grid;
+  align-items: center;
+  justify-content: center;
+  background: rgba(0, 0, 0, 0.1);
+  ::ng-deep .mat-spinner circle {
+    stroke: mat-color($prx-gray, 100);
+  }
+}
+
+.empty {
+  height: 80px;
+  display: grid;
+  align-items: center;
+  justify-content: center;
+  color: mat-color($prx-gray, 400);
+}
+
+.row {
```

Comment:
This is a bit duplicative of the inventory-table css ... but it's non trivial to share that between components, or at least i don't know a great way.  Ideally, we'd convert these all to Material tables, and restyle from there.